topics/application-gradle-plugin.md

```diff
@@ -0,0 +1,68 @@
+[//]: # (title: Distribution via Gradle Application plugin)
+
+From the Gradle [documentation](https://docs.gradle.org/current/userguide/application_plugin.html):
+> The Application plugin facilitates creating an executable JVM application. 
+> It makes it easy to start the application locally during development, 
+> and to package the application as a TAR and/or ZIP including operating system specific start scripts.
+
+We will use the second responsibility of this plugin and show how to package a simple Ktor application.
+
+Let's start with the following project structure:
+
+```text
+├── build.gradle
```

Comment:
Does this reflect the actual distribution that is being used? Cause I see three Gradle files.

Reply:
It's the initial project structure to start with

Reply:
> Let's start with the following project structure

What does this mean? I should generate an empty Gradle project by myself? Why `snippets/gradle-app` is not a runnable project and doesn't reflect this structure?